### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by EthanStraub**
- Fixed a bug where swears weren't being detected in messages that started in mentions
- Added in some more swears

### CODE REVIEW

File: client.py [hunk 1]
```diff
@@ -1232,20 +1236,54 @@
...
+
+					usermodel.swear_jar += 1
+
+			market_data.persist()
+
 			if usermodel != None:
-				usermodel.swear_jar += 1
-				usermodel.slimecoin = max(0, usermodel.slimecoin - 1000000000) # 1 billion slimecoin fine for swearing
+					
+				usermodel.slimecoin = max(0, (usermodel.slimecoin - (usermodel.swear_jar * swear_multiplier * 10000))) # fine the user for swearing, 1 billion slimecoin for every swear in their swear jar
```
Comment (magicksRampage) [inline client.py]: we should use change_slimecoin() for this and track how much each player has actually been fined over their lifetime

File: ewapt.py [hunk 2]
```diff
@@ -996,6 +992,12 @@ def check(str):
 							accepted = True
 						if message.content.lower() == ewcfg.cmd_refuse:
 							accepted = False
+							
+							# Flag the person knocking to discourage spam
+							user_data.time_expirpvp = ewutils.calculatePvpTimer(user_data.time_expirpvp, (int(time.time()) + ewcfg.time_pvp_knock))
```
Comment (magicksRampage) [inline ewapt.py:999]: user_data hasn't been reloaded since the last await

**Review by magicksRampage — CHANGES_REQUESTED** [review]

File: client.py [hunk 3]
```diff
@@ -1275,8 +1275,8 @@
 			market_data.persist()
 
 			if usermodel != None:
-					
-				usermodel.slimecoin = max(0, (usermodel.slimecoin - (usermodel.swear_jar * swear_multiplier * 10000))) # fine the user for swearing, 1 billion slimecoin for every swear in their swear jar
+				# fine the user for swearing, based on how much they've sworn right now, as well as in the past
+				usermodel.change_slimecoin(n= -1 * (usermodel.swear_jar * swear_multiplier * 10000), coinsource=ewcfg.coinsource_swearjar)
```
Comment (magicksRampage) [inline client.py]: change_slimecoin doesn't actually make it so value bottoms out at 0. you can have negative slimecoin this way. though that may be funny in its own way

Comment (EthanStraub) [reply client.py]: I don't even wanna fucking think about what could happen as a result of that, so I'll go ahead and fix this.

**Pull request merged**